sweepai/utils/docker_utils.py

```diff
@@ -0,0 +1,33 @@
+import requests
+import json
+import subprocess
+
+def get_docker_hub_version():
+    response = requests.get('https://hub.docker.com/v2/repositories/sweepai/sweep/tags')
```

Comment:
can you use the "last_pushed": "2023-10-02T06:18:25.7857Z" field on both local and the requests call?